.github/workflows/build.yml

```
@@ -14,7 +14,7 @@ jobs:
   build:
     strategy:
       matrix:
-        node: [14, 16]
+        node: [18, 16]
     name: Build Packages
```

```suggestion
    name: Build CJS
```
lets be consistent with other build naming and more visibility in ci ?

---

done